products/memgraph/upcoming/tutorials/football-transfers.md

```diff
@@ -0,0 +1,390 @@
+RETURN ROUND(max_fee) + 'M €' as max_transfer_fee, season_name
+ORDER BY max_fee DESC
+```
+
+3) How about finding out which teams your favorite player has played for?
+If you wish to check the teams for another player, replace "Sime Vrsaljko"
+with the name of your favorite player.
+
+```opencypher
+MATCH
+    (player:Player)-[:TRANSFERRED_IN]->(t:Transfer)-[]-(team:Team)
+WHERE
+    player.name = "Sime Vrsaljko"
+WITH DISTINCT team
+RETURN team.name AS team_name
+```
+
+You might wonder why we haven't specified a direction in our Cypher traversal with `(:Transfer)-[]-(:Team)`.
+As we want to find the teams that player was transferred from (`(:Transfer)<-[]-(:Team)`) and transferred to
+(`(:Transfer)-[]->(:Team)`), we want to collect both inbound and outbound connections. In order to do so, we
+omit the arrow (`>`, `<`) in our Cypher command.
+
+4) Find players that were transferred to and played for FC Barcelona and
+count them by the player game position.
+
+```opencypher
+MATCH
+    (team:Team)<-[:TRANSFERRED_TO]-(t:Transfer)<-[:TRANSFERRED_IN]-(player:Player)
+WHERE
+    team.name = "FC Barcelona"
+WITH DISTINCT player
+RETURN player.position as player_position, COUNT(player) AS position_count, collect(player.name) as player_names
+ORDER BY position_count DESC
+```
+
+5) Football has seen a lot of rivalries develop between clubs during its rich and long history.
+One of the most famous ones is between fierce rivals FC Barcelona and Real Madrid.
+There is a term, El Clasico, for a match between those two teams. Let's find all the transfers between
+FC Barcelona and Real Madrid.
+
+```opencypher
+MATCH
+    (m:Team)-[:TRANSFERRED_FROM]-(t:Transfer)-[:TRANSFERRED_TO]-(n:Team),
+    (t)<-[:TRANSFERRED_IN]-(p:Player)
+WHERE
+    (m.name = "FC Barcelona" AND n.name = "Real Madrid") OR
+    (m.name = "Real Madrid" AND n.name = "FC Barcelona")
+RETURN m.name as transferred_from_team, p.name as player_name, n.name as transfered_to_team
+```
+
+6) FC Barcelona is one of the most valuable football clubs in the world. Players often want to play there as long as possible.
+But what about those players who didn't fit in well? Where do they go?
+
+```opencypher
+MATCH
+    (m:Team)-[:TRANSFERRED_FROM]->(t:Transfer)<-[:TRANSFERRED_IN]-(p:Player),
+    (t)-[:TRANSFERRED_TO]->(n:Team)
+WHERE
+    m.name = "FC Barcelona"
+RETURN n.name as team_name, collect(p.name) as player_names, COUNT(p) AS number_of_players
+ORDER BY number_of_players DESC
+```
+
+7) What are the teams that most players went to in season 2003/2004? The results may surprise you.
+
+```opencypher
+MATCH
+    (season:Season)<-[:HAPPENED_IN]-(t:Transfer)<-[:TRANSFERRED_IN]-(player:Player),
+    (t)-[:TRANSFERRED_TO]->(team:Team)
+WHERE
+    season.name = "2003/2004"
+WITH DISTINCT player, team
+RETURN team.name as team_name, COUNT(player) AS number_of_players, collect(player.name) as player_names
+ORDER BY number_of_players DESC, team_name
+LIMIT 20;
```

Comment:
Remove the `;` from the end of this line.

Reply:
:+1: